pkg/controllers/dashboard/fleetcharts/controller.go

```diff
@@ -107,6 +110,14 @@ func (h *handler) onSetting(key string, setting *v3.Setting) (*v3.Setting, error
 		gitjobChartValues["noProxy"] = envVal
 	}
 
+	// add priority class value
+	if priorityClassName, err := chart.GetPriorityClassNameFromRancherConfigMap(h.configCache); err != nil {
+		logrus.Warnf("Failed to get rancher priorityClassName: %v", err)
```

Comment:
I wasn't positive how this would shake out in all update cases since this would now require users to have a rancher config map. I'll update and we can verify an upgrade case while testing.

Reply:
Turn's out returning an error breaks a lot of things. If we can't install our required charts without the configMap being present, we can not run rancher standalone without using helm, which is a significant breaking change that is not necessary.

An alternative would be to add a controller that creates the configMap for us if it doesn't exist, But I am not sure we want to go down that road just yet since priorityClassName is not critical functionality.